src/context/DataContext.tsx

```diff
@@ -9,6 +9,8 @@ export const DataContext = createContext<DataContextValueType>({
   },
 });
 
+export const useDataContext = () => useContext(DataContext);
```

Comment:
I don't think we need this context, this can be a simple hook. Also, the library already have a hook that is doing the same (useAsyncStorage). I'll create an issue to refactor this

Reply:
Issue: #158